### DISCUSSION TITLE
German keyboard: Underscore "_" as second assignment.

### DISCUSSION THREAD

**Discussion by samarutuk** [florisboard/florisboard#1729] [Feedback]
Hi, I'm talking about the German keyboard layout with keys for the umlauts (ä, ö, ü) plus the "ß" character. Unfortunately, the underscore  "_" is not easily accessible as a second assignment. Since it is used for italic text, easy access would be important. The character "ü" is still without a second assignment. BTW, the backslash \ would not be bad as a second assignment, either.

**Comment by cboehm-it** [comment]
I really support that. I need the underscore very often. Maybe I need to move to another keyboard. 😞
